p4-16/psa/PSA.mdk

```diff
@@ -388,15 +388,14 @@ to raise an error.
 
 For RESUB packets, `packet_in` is the same as the pre-IngressParser
 contents of `packet_in`, for the packet that caused this resubmitted
-packet to occur (i.e. with NO modifications from any ingress processing).  Truncation is not supported for RESUB packets.
+packet to occur (i.e. with NO modifications from any ingress processing).
```

Comment:
Note that removing this sentence 'Truncation is not supported for RESUB packets.' does _not_ mean that truncation is supported in PSA for resub packets -- with this PR, truncation is _only mentioned as an option for cloned packets_.  If we don't say anything about truncation for other kinds of packets, then we are neither prohibiting it nor requiring it for those other kinds of packets.